frontend/www/js/omegaup/components/problem/FinderWizard.vue

```diff
@@ -228,7 +234,7 @@ export default class ProblemFinderWizard extends Vue {
       queryParameters.only_karel = true;
     }
     if (this.selectedTags.length > 0) {
-      queryParameters.tag = this.selectedTags;
+      queryParameters.tag = this.selectedTags.map(tag => tag.key);
```

Comment:
```suggestion
      queryParameters.tag = Object.keys(this.selectedTags);
```

si entiendo bien, esto debería evitar los demás cambios de `TagsObject` => `TagObject`

Reply:
Me parece que no. Cambié el tipo de `tagsObject` porque ahora `vue-tags-input` recibe un arreglo de objetos, lo cual me permite mantener el `key` para la búsqueda y el `value` para lo que quiero que vea el usuario.

`Object.keys(this.selectedTags)` no funcionaría, pues `this.selectedTags` es un arreglo, no un objeto.

Reply:
Por esa razón también actualicé la biblioteca de ese componente.

Reply:
oh ya, entonces esto está bien.